scripts/e2e/auth-claude-login.sh

```
@@ -0,0 +1,142 @@
+}
+
+trap 'on_error $LINENO' ERR
+trap cleanup EXIT
+
+command -v timeout >/dev/null 2>&1 || fail "missing 'timeout' command"
+
+dg_ensure_docker "$ROOT/.e2e-bin"
+dg_prepare_docker_git_cli "$REPO_ROOT" "$ROOT/.e2e-bin"
+
+set +e
+timeout "${LOGIN_TIMEOUT_SECONDS}s" bash -lc 'cd "$1" && bun packages/app/dist/src/docker-git/main.js auth claude login' bash "$REPO_ROOT" \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)